src/parachain/redeem.ts

```diff
@@ -42,10 +41,10 @@ export function encodeRedeemRequest(req: RedeemRequest, network: Network, id: H2
     return {
         id: stripHexPrefix(id.toString()),
         userDOTAddress: req.redeemer.toString(),
-        amountBTC: satToBTC(req.amount_btc).toString(),
-        dotPremium: planckToDOT(req.premium).toString(),
-        bridgeFee: satToBTC(req.fee).toString(),
-        btcTransferFee: satToBTC(req.transfer_fee_btc).toString(),
+        amountBTC: BTCAmount.from.Satoshi(req.amount_btc.toString()).str.BTC(),
```

Comment:
What is `.str`?

Reply:
It's a field of Monetary.js objects that has methods for displaying the amount in all denominations (e.g. for Bitcoin, `.str.BTC()` and `.str.Satoshi()`)